services/app-web/src/pages/StateSubmission/RateDetails/RateDetails.test.tsx

```diff
@@ -853,198 +852,6 @@ describe('RateDetails', () => {
                 expect(continueButton).toHaveAttribute('aria-disabled', 'true')
             })
         })
-
-        it('progressively disclose new rate form fields on the second rate', async () => {
```

Comment:
I deleted some unnecessary and long running tests. For example this logic is handled with the single rate test, there isn't a need to do the same test for multiple rates since its all shared logic and components at a high level.